### PR TITLE
Accepting removal API and increasing upgrade timeout period

### PR DESCRIPTION
**Pull request by Santhoshairody**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Santhoshairody** [comment]
Test logs here:
https://jenkins.pwx.dev.purestorage.com/job/Torpedo/view/Torpedo%20NextPX/job/tp-ocp-upgrade/

File: drivers/scheduler/openshift/openshift.go [hunk 1]
```diff
@@ -576,7 +576,7 @@ func waitNodesToBeReady() error {
 		return nil, false, nil
 	}
 
-	_, err = task.DoRetryWithTimeout(t, 30*time.Minute, 15*time.Second)
+	_, err = task.DoRetryWithTimeout(t, 600*time.Minute, 30*time.Second)
```
Comment (lsrinivas-pure) [inline drivers/scheduler/openshift/openshift.go:579]: does it required 10 hours for nodes to be ready?

File: drivers/scheduler/openshift/openshift.go [hunk 2]
```diff
@@ -725,11 +725,17 @@ func ackAPIRemoval(version string) error {
 	}
 	// this issue happens on OCP 4.9
 	parsedVersion49, _ := semver.Parse("4.9.0")
-
+	parsedVersion412, _ := semver.Parse("4.12.0")
 	if parsedVersion.GTE(parsedVersion49) {
+		var patchData = ""
```
Comment (nikolaypopov) [inline drivers/scheduler/openshift/openshift.go:730]: make this `var patchData string`

**Comment by Santhoshairody** [comment]
New PR is here
https://github.com/portworx/torpedo/pull/1535

**Pull request closed without merging**